app/main.py

```diff
@@ -1,8 +1,20 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: str) -> None:
+        self.name = name
+        self.age = age
+        self.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+    for line in people:
+        person_list.append(Person(line["name"], line["age"]))
+
+    for obj in people:
```

Comment:
use more informable variables for loop, such as person in people

Reply:
fixed. I'm understood, thank you